oses/Dockerfile.fedora31

```diff
@@ -13,10 +13,10 @@ RUN dnf install -y autoconf automake \
                         Judy-devel \
                         lm_sensors \
                         make \
-                        MySQL-python \
                         nc \
                         pkgconfig \
                         python \
+                        python-mysql \
                         python-psycopg2 \
```

Comment:
why do we need `lm_sensors`, `python-mysql`, `python-psycopg2`, etc?

Reply:
I honestly have no idea. I suspect the SQL stuff is so that we (the developers) can build the images locally for testing of such things, but I don't know for certain. Almost everything here was done by @paulkatsoulakis, and the PR's and commits are somewhat distressingly devoid of reasoning regarding such things.